f2/src/forms/InformationForm.js

```diff
@@ -10,11 +10,22 @@ import { Stack, FormControl, VisuallyHidden } from '@chakra-ui/core'
 import { FormHelperText } from '../components/FormHelperText'
 import { FormLabel } from '../components/FormLabel'
 import { P } from '../components/paragraph'
+import { useStateValue } from '../utils/state'
+
+export const InformationForm = props => {
+
+  const [data] = useStateValue ()
+  const information = {
+    information:'',
+    ... data.formData.information,
```

Comment:
Oh! and more important: the submit function on the InformationPage is saving to `data.formData.personalInformation`, not `data.formData.information`.